geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1860,7 +1860,10 @@
    * <U>Default</U>: <code>""</code>
    * </p>
    * <U>Since</U>: GemFire 8.0
+   *
+   * @deprecated Since Geode 1.4 (See GEODE-3007)
    */
+  @Deprecated()
```

Comment:
All uses of this property have been removed in this change - is that going to be a breaking change anywhere?

Reply:
No, because the property itself isn't gone.